cs6232-g4/Controller/FurnitureController.cs

```diff
@@ -35,5 +35,20 @@ public void UpdateFurniture(int furnitureId, int rentedQuantity)
         {
              this._furnitureDAL.UpdateFurniture(furnitureId, rentedQuantity);
         }
+
+        internal List<Furniture> GetFurnitureByCategory(string category)
```

Comment:
this isn't implemented. It needs to get the furniture from the database. Check how MembersController/MemberDAL implemented for an example

Reply:
Should be resolved now.